backend-checks/src/main/scala/uk/gov/nationalarchives/tdr/localaws/backendchecks/checks/FileFormatCheck.scala

```diff
@@ -0,0 +1,26 @@
+package uk.gov.nationalarchives.tdr.localaws.backendchecks.checks
+
+import java.util.UUID
+
+import uk.gov.nationalarchives.tdr.localaws.backendchecks.api.FileService
+import uk.gov.nationalarchives.tdr.localaws.backendchecks.auth.TokenService
+import uk.gov.nationalarchives.tdr.localaws.backendchecks.checks.metadata.{FakeAntivirusMetadata, FakeFileFormat}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class FileFormatCheck(
+                       tokenService: TokenService,
+                       fileService: FileService
+                     )(implicit val executionContext: ExecutionContext) extends FileCheck {
+
+  override def name: String = "antivirus"
```

Comment:
Should the name be "fileFormat" and not "antivirus"

Reply:
Copypasta! Fixed, thanks.